include/flatbuffers/flatbuffers.h

```diff
@@ -473,26 +474,22 @@ class Array {
 
   // Similarly, but typed, much like std::vector::data
   const T *data() const { return reinterpret_cast<const T *>(Data()); }
-  T *data() {
-    #if defined(FLATBUFFERS_ALIGN_CHECK) && !defined(FLATBUFFERS_CPP98_STL)
-    // The data_ is aligned as this Array<T,length>.
-    // But this Array<T,length> themself must be aligned not stricker than T.
-    // Flatbuffers uses reinterpret_cast from T-aligned memory data_ to a
-    // reference of the Array<T, lenght>.
-    static_assert(
-        alignof(std::remove_reference<decltype(*this)>::type) <= alignof(T),
-        "Invalid alignment");
+  T *data() { return reinterpret_cast<T *>(Data()); }
 
-    // The idl_gent_text.cpp uses reintepret_cast from the data_ memory to
-    // Array<Offset<void>, lenght>.
-    static_assert(alignof(Array<Offset<void>, length, false>) <= alignof(T),
-                  "Invalid alignment");
-    #endif
+ protected:
+  void MutateImpl(flatbuffers::integral_constant<bool, true> tag, uoffset_t i,
```

Comment:
Remove `tag` name and cast to void below?

Reply:
Yes, you are right.